code401Challenges/src/test/java/code401Challenges/linkedlist/LinkedListTest.java

```diff
@@ -56,4 +53,82 @@ public void testPrint(){
         assertEquals(expectedOutput, classUnderTest.print());
     }
 
-}
\ No newline at end of file
+    @Test
+    public void testAppend(){
+        LinkedList classUnderTest = new LinkedList();
+        int num = 10;
+        classUnderTest.insert(15);
```

Comment:
Instead of using `insert` and then only using `append` at the end, it would have been better to just use `append`